### PR TITLE
SDK-2082: Add identity profile report attribute retrieval

### PR DESCRIPTION
**Pull request by irotech**

### CODE REVIEW

File: yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java [hunk 1]
```diff
@@ -193,4 +193,14 @@ private void findAllAgeVerifications() {
         return (Attribute<List<Image>>) a;
     }
 
+    /**
+     * JSON object containing identity assertion and the verification report.
+     * This will be null if not provided by Yoti.
+     *
+     * @return the identity profile report
+     */
+    public Attribute<Map<String, Object>> getIdentityProfileReport() {
```
[the remark this answers is not in the document]
Comment (BenSmithYoti) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java:202]: If my understanding is correct (?) - then I think this is best left as is, it seems clearer to me.

**Review by BenSmithYoti — APPROVED** [review]
Looks good to me. Thanks vince.

File: yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (BenSmithYoti) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java:202]: I think this is good as it is. All seems sensible to me.

[the remark this answers is not in the document]
Comment (pn-santos) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java:202]: `?` and `Object` are equivalent. Usually the reason to use `?` is when there's a need to restrict the type to `? extends X`

**Review by pn-santos — APPROVED** [review]

**Review by echarrod — APPROVED** [review]

**Pull request merged**